app/src/main/java/org/schabi/newpipe/settings/ContentSettingsMainDialog.java

```diff
@@ -0,0 +1,123 @@
+package org.schabi.newpipe.settings;
+
+import android.app.Dialog;
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.content.res.TypedArray;
+import android.graphics.Canvas;
+import android.graphics.drawable.Drawable;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.DialogFragment;
+import android.support.v7.widget.CardView;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+
+import org.schabi.newpipe.R;
+import org.schabi.newpipe.database.subscription.SubscriptionEntity;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class ContentSettingsMainDialog extends DialogFragment {
```

Comment:
Remove this whole class and use `AlertDialog` instead, like [here](https://github.com/teamnewpipe/NewPipe/blob/38946e4b0f28501fd357b3d4f894646aa7d49146/app/src/main/java/org/schabi/newpipe/info_list/InfoItemDialog.java#L45).